py_hamt/sharded_zarr_store.py

```
@@ -145,17 +289,21 @@ class ShardedZarrStore(zarr.abc.store.Store):
+    path, allowing grouped arrays to reuse chunk coordinates without collisions.
     """
 
+    _V1_COORDINATE_ARRAY_PREFIXES = {
+        "time",
+        "lat",
+        "lon",
+        "latitude",
+        "longitude",
+        "forecast_reference_time",
+        "step",
+    }
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)